packages/react-scripts/template/README.md

```diff
@@ -2048,6 +2049,18 @@ Add the following scripts in your `package.json`:
 
 The `predeploy` script will run automatically before `deploy` is run.
 
+If you are deploying to a GitHub user page instead of a project page you'll need to make two
+additional modifications:
+
+1. First, change your repository's source branch to be any branch other than **master**.
```

Comment:
Is this the only way to accomplish this? Seems kind of hacky but I don't know enough about GitHub Pages.

Reply:
Taken from GitHub settings for my particular user page: "User pages must be built from the master branch."

Unfortunately, I don't see any other way around this. At least until GitHub decides to support hosting user pages out of other branches besides master.

Reply:
Great, that's all the confirmation I needed. 😄